File: fizzbuzz/FizzBuzz.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
  for(int i = 1;true; i++) {
  		string output = "";
        if(i % 3 == 0) {
        	output += "Fizz";
        } 
        if(i % 5 == 0) {
        	output += "Buzz";
        }
        if(output == "") 
        {
			output = std::to_string(i);
        }
        cout << output << '\n';
  }
  return 0;
}

File: fizzbuzz/FizzBuzz.php
<?php
for($counter = 1; true; $counter++) {
    $output = "";
    if($counter % 3 == 0) {
        $output .= "Fizz";
    }
    if($counter % 5 == 0) {
        $output .= "Buzz";
    }
    if($output = "") {
        $output = "$counter";
    }
    echo $output
}
?>